charts/celeborn/templates/master-statefulset.yaml

```diff
@@ -46,7 +46,7 @@ spec:
       labels:
         app.kubernetes.io/name: {{ .Chart.Name }}
         app.kubernetes.io/instance: {{ .Release.Name }}
-        app.kubernetes.io/version: {{ .Values.image.tag | default .Chart.AppVersion | quote }}
+        app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
```

Comment:
We may add a user custom label of image tag in statefulset template. As we can know which image tag the pod from the labels.

Reply:
Sure, please review the modified.